x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/LookupJoinTypesIT.java

```diff
@@ -265,6 +268,21 @@ private static boolean existingIndex(Collection<TestConfigs> existing, DataType
         return existing.stream().anyMatch(c -> c.exists(indexName));
     }
 
+    public void testOutputSupportedTypes() throws Exception {
```

Comment:
nit: maybe a comment that this is used to generate a table of the supported types?